core/primitives/src/version.rs

```diff
@@ -155,7 +155,7 @@ pub const PEER_MIN_ALLOWED_PROTOCOL_VERSION: ProtocolVersion = MAIN_NET_PROTOCOL
 /// Current protocol version used on the main net.
 /// Some features (e. g. FixStorageUsage) require that there is at least one epoch with exactly
 /// the corresponding version
-const MAIN_NET_PROTOCOL_VERSION: ProtocolVersion = 51;
+const MAIN_NET_PROTOCOL_VERSION: ProtocolVersion = 52;
```

Comment:
Unrelated to this PR but the naming confused me: why mainnet specifically?

Reply:
Was introduced in https://github.com/near/nearcore/pull/5610/. I *think* it's indeed better to rename it to `STABLE_PROTOCOL_VERSION`. We need this for `PEER_MIN_ALLOWED_PROTOCOL_VERSION`, to allow nightly peers to connect to mainnet I think.

Reply:
Agree that `STABLE_PROTOCOL_VERSION` makes a lot more sense